config/sources/families/rockchip.conf

```diff
@@ -56,7 +58,7 @@ uboot_custom_postprocess()
 	# so reset works more reliably. U-boot image is set to be at sector 0x200 on the eMMC/SD, 
 	# so we burn it 0x200-0x40 because of the rockchip 0x40 sectors offset.
 	if [[ $BOARD == xt-q8l-v10 ]]; then
-		tools/mkimage -n rk3288 -T rksd -d $SRC/cache/sources/rkbin-tools/rk32/rk3288_ddr_400MHz_v1.03.bin u-boot-rockchip-with-spl.bin
+		tools/mkimage -n rk3288 -T rksd -d $SRC/packages/blobs/rockchip/rk3288_ddr_400MHz_v1.08.bin u-boot-rockchip-with-spl.bin
```

Comment:
Why not putting the file into https://github.com/armbian/rkbin with other Rockchip blobs?

Reply:
I took this from Kwiboo's rkbin repository (https://github.com/Kwiboo/rkbin/tree/libreelec-cleanup/rk32). But I don't know if it is an "official" ddrbin or not.
If you think it is ok to push this into official armbian rkbin I will do that and fix this reference.